server/channels/api4/preference_test.go

```diff
@@ -623,6 +662,28 @@ func TestDeletePreferences(t *testing.T) {
 	resp, err = client.DeletePreferences(context.Background(), th.BasicUser.Id, preferences)
 	require.Error(t, err)
 	CheckUnauthorizedStatus(t, resp)
+
+	//DeletePreferences API from System Admin and Local Client
+	th.TestForSystemAdminAndLocal(t, func(t *testing.T, c *model.Client4) {
+
+		//Creating Test Data
+		var preferences model.Preferences
+		preference := model.Preference{
+			UserId:   th.BasicUser.Id,
+			Category: model.PreferenceCategoryCustomStatus,
+			Name:     model.NewId(),
+			Value:    "true",
+		}
+		preferences = append(preferences, preference)
+		c.UpdatePreferences(context.Background(), th.BasicUser.Id, preferences)
+
+		//Delete Prefrerences Operation
+		resp, err = c.DeletePreferences(context.Background(), th.BasicUser.Id, preferences)
+
+		//Validations
```

Comment:
Nit: that is a bit too verbose

```suggestion
		resp, err = c.DeletePreferences(context.Background(), th.BasicUser.Id, preferences)
```

Reply:
Fixed